ambry-tools/src/integration-test/java/com.github.ambry/clustermap/HelixBootstrapUpgradeToolTest.java

```diff
@@ -87,24 +94,68 @@ public static void initialize() throws IOException {
     }
   }
 
+  @Parameterized.Parameters
+  public static List<Object[]> data() {
+    return Arrays.asList(new Object[][]{{"DC1"}, {"DC1, DC0"}, {"all"}});
```

Comment:
I feel like test state has been persisted in cluster. That is, if we change the order, like move `{"DC1"}` to the end. `testEverything` will fail. Can you fix this?